resources/server/bin/helpers/check-requirements-linux.sh

```diff
@@ -81,7 +81,13 @@ if [ -n "$(ldd --version | grep -v musl)" ]; then
 		# Rather than trusting the output of ldd --version (which is not always accurate)
 		# we instead use the version of the cached libc.so.6 file itself.
         libc_real_path=$(readlink -f "$libc_path")
-        libc_version=$($libc_real_path --version | sed -n 's/.*stable release version \([0-9]\+\.[0-9]\+\).*/\1/p')
+        if [ -x "$libc_real_path" ]; then
+            # get version from executable
+            libc_version=$($libc_real_path --version | sed -n 's/.*stable release version \([0-9]\+\.[0-9]\+\).*/\1/p')
+        else
+            # .so is not executable on this host; try getting from strings
+            libc_version=$(strings "$libc_real_path" | sed -n 's/.*stable release version \([0-9]\+\.[0-9]\+\).*/\1/p')
```

Comment:
strings might not be available by default in most scenarios

```suggestion
            libc_version=$(cat "$libc_real_path" | sed -n 's/.*stable release version \([0-9]\+\.[0-9]\+\).*/\1/p')
```